Apps/phtaniumdetect/taniumdetect.json

```diff
@@ -0,0 +1,2261 @@
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.*.name",
+                    "data_type": "string",
+                    "column_name": "Name",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.data.*.*.description",
+                    "data_type": "string",
+                    "column_name": "Description",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.data.*.*.intelDocId",
+                    "data_type": "numeric",
+                    "column_name": "Intel Document ID",
+                    "column_order": 3
+                },
+                {
+                    "data_path": "action_result.data.*.*.updatedAt",
+                    "data_type": "string",
+                    "column_name": "Updated At",
+                    "column_order": 6
+                },
+                {
+                    "data_path": "action_result.data.*.*.id",
+                    "data_type": "numeric",
+                    "column_name": "ID",
+                    "column_order": 2
+                },
+                {
+                    "data_path": "action_result.data.*.*.createdAt",
+                    "data_type": "string",
+                    "column_name": "Created At",
+                    "column_order": 5
+                },
+                {
+                    "data_path": "action_result.data.*.*.revision",
+                    "data_type": "numeric",
+                    "column_name": "Revision",
+                    "column_order": 4
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "delete suppression rule",
+            "identifier": "delete_suppression_rule",
+            "description": "Delete one or more suppression rules",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "id": {
+                    "description": "The ID of the rule to delete, Multiple occurrences are supported",
```

Comment:
Please specify how the user should provide multiple IDs. Is it a comma-separated list?

Reply:
@robdrouin @cwadhwani-crest Is it a comma-separated list?

Reply:
Whatever input is provided in the ID parameter it will be passed as id = <input_from_the_user> in the params section of the request. As we are not having the asset we are not sure if the API takes comma-separated values or not. Therefore, keeping it as it is.